{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "5e72c581_4b209add",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1247893
      },
      "writtenOn": "2023-04-19T09:25:30Z",
      "side": 1,
      "message": "The testing failure looks rather odd here (and doesn\u0027t appear to be related to the patch) - could you take a look?",
      "revId": "7b7f84fe148168532bbf9add7b738d125588c926",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}